### ISSUE TITLE
Reporting Issue

### ISSUE THREAD

**Issue by slyfox29** [face-hh/lyntr#45]
There report profile and report lynt future is not working. also maybe adding like four or five common reporting selections would be good because for simple things 50 characters is hard to get

**Comment by GamplaxPramgrammer** [comment]
Same here! I am trying my best to report something, but it always gives me something like "Failed to submit report. Please try again."

**Comment by ironland-web** [comment]
Yep, same thing; i tried to report the spam accounts.

**Comment by sweglord227** [comment]
true. a drop down select would be nice for more common stuff like spam, with an other + text box for more complicated things
